### PR TITLE
:sparkles: Set application delete as danger kebab menu item (#1829)

### PR DESCRIPTION
**Pull request by sjd78**
Part of #1318
Resolves: https://issues.redhat.com/browse/MTA-2261
Backport-of: #1829

### CODE REVIEW

File: client/src/app/pages/applications/applications-table/applications-table.tsx [hunk 1]
```diff
@@ -1010,26 +1010,6 @@ export const ApplicationsTable: React.FC = () => {
...
-                                    isAriaDisabled:
-                                      application.migrationWave !== null,
-                                    tooltipProps: {
-                                      content:
-                                        application.migrationWave !== null
-                                          ? t(
-                                              "message.cannotDeleteApplicationsAssignedToMigrationWave"
-                                            )
-                                          : "",
-                                    },
```
Comment (mguetta1) [inline client/src/app/pages/applications/applications-table/applications-table.tsx:1025]: @sjd78 did you delete lines 1013-1025 in purpose?
If the reason is to backport also #1803, please modify `applicationDropdownItems` as well

**Review by mguetta1 — CHANGES_REQUESTED** [review]

File: client/src/app/pages/applications/applications-table/applications-table.tsx (hunk 1 above)
Comment (mguetta1) [reply client/src/app/pages/applications/applications-table/applications-table.tsx:1025]: Looks like #1803 should not be backported yet, fix version is MTA 7.1.0
https://issues.redhat.com/browse/MTA-2225

Comment (sjd78) [reply client/src/app/pages/applications/applications-table/applications-table.tsx]: Removed this erroneous change in the latest push.

Comment (sjd78) [reply client/src/app/pages/applications/applications-table/applications-table.tsx:1025]: Oh wait -- I see the nuance now.  I'll keep the pre-1803 version of the delete button but move it to the newer location, and add the danger style

**Review by mguetta1 — APPROVED** [review]
LGTM

**Pull request merged**